packages/api-plugin-fulfillment/src/mutations/updateFulfillmentType.js

```diff
@@ -0,0 +1,60 @@
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+
+const inputSchema = new SimpleSchema({
+  fulfillmentGroupId: String,
+  shopId: String,
+  name: String,
+  enabled: {
+    type: Boolean,
+    defaultValue: true
+  },
+  label: String,
+  displayMessageType: {
+    type: String,
+    optional: true
+  }
+});
+
+/**
+ * @method updateFulfillmentType
+ * @summary updates the selected fulfillment type
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - Input object
+ * @param {String} input.fulfillmentGroupId - fulfillment tpe id of group
+ * @param {String} input.shopId - Shop Id
+ * @param {String} input.name - name of fulfillment type
+ * @param {Boolean} input.enabled - status of ff-type
+ * @param {String} input.label - label of ff-type
+ * @param {String} input.displayMessageType - optional display message for UI
+ * @returns {Promise<Object>} An object with the updated type
+ */
+export default async function updateFulfillmentType(context, input) {
+  const cleanedInput = inputSchema.clean(input); // add default values and such
+  inputSchema.validate(cleanedInput);
+
+  const { fulfillmentGroupId, shopId, name, enabled, label, displayMessageType } = cleanedInput;
```

Comment:
`fulfillmentGroupId` should be `fulfillmentTypeId`

Reply:
updated